containers/e2b-sandbox/README.md

```diff
@@ -0,0 +1,15 @@
+# How to build custom E2B sandbox for OpenDevin
+
+[E2B](https://e2b.dev) is an [open-source](https://github.com/e2b-dev/e2b) secure cloud environment (sandbox) made for running AI-generated code and agents. E2B offers [Python](https://pypi.org/project/e2b/) and [JS/TS](https://www.npmjs.com/package/e2b) SDK to spawn and control these sandboxes.
+
+
+1. Install the CLI with NPM.
+    ```sh
+    npm install -g @e2b/cli@latest
+    ```
+    Full CLI API is [here](https://e2b.dev/docs/cli/installation).
+
+1. Build the sandbox
+  ```sh
+  e2b template build --dockerfile ./Dockerfile --name "open-devin"
```

Comment:
Can we `e2b template` from a remote image like `ubuntu` or `ghcr.io/opendevin/sandbox`? Or does the user have to build it from scratch?

Would be nice not to have to maintain a new Dockerfile, or to at least build it for them in our CI/CD pipeline

Reply:
You can share the same Dockerfile and use a remote image. We just have some restrictions on what images and Dockerfile commands we support. Generally, you should be good. I'll set it up to use the same Dockerfile image

<img width="943" alt="Screenshot 2024-04-16 at 2 53 10 PM" src="https://github.com/OpenDevin/OpenDevin/assets/5136688/b01d4b2a-6f3f-4a31-8dbe-d6e41ac0b2ec">